app.py

```diff
@@ -1,23 +1,35 @@
 #!/usr/bin/env python3
-import random
-import string
+# Installed
+from aws_cdk import App, Environment
+# Local
+from sds_data_manager.utils.stackbuilder import build_sds
 
-import aws_cdk as cdk
+"""
+This app is designed to be the dev and production deployment app. It defaults to a dev deployment 
+via a default `deploy-where` value in cdk.json. To deploy to prod, specify `--context deploy-where=prod`.
```

Comment:
I like this! If this isn't a cdk term, might I suggest "env" or "environment" instead of "deploy-where"?

Reply:
That makes sense. Changed it.